mc-sema/cfgToLLVM/x86Instrs_MOV.cpp

```diff
@@ -573,6 +573,14 @@ static InstTransResult translate_MOVao (NativeModulePtr natM, BasicBlock *& bloc
     Function *F = block->getParent();
     Module *M = F->getParent();
 
+    // this is awful, but sometimes IDA detects the immediate
+    // as a memory reference. However, this instruction can only
+    // have an immediate, so this is safe
+    if( ip->has_imm_reference || ip->has_mem_reference ) {
```

Comment:
are there times when an imm ref is not a mem ref? how will this interact with binja? What I'm really asking is: are the semantics implied in the protobuf consistent with how they are interpreted in the code? is there a corresponding comment in the protobuf describing this case?